Object in array in arguments attribute fails to compile

schema.graphql:

```graphql
schema {
    query: Query
}

input Args {
    field: Int
}

type Query {
    my_query(args: [Args]): Int!
}
```

main.rs:
```rs
#[cynic::schema("my_schema")]
mod schema {}

#[derive(cynic::QueryFragment)]
#[cynic(graphql_type = "Query")]
struct MyQuery {
    #[arguments(args: [{field: 1}])]
    #[cynic(rename = "my_query")]
    my_query: i32
}

fn main() {
    println!("Hello, world!");
}
```

Compiler output:
```
error[E0308]: mismatched types
 --> src/main.rs:4:10
  |
4 | #[derive(cynic::QueryFragment)]
  |          ^^^^^^^^^^^^^^^^^^^^ expected `()`, found `ObjectArgumentBuilder<'_, Args, ()>`
  |
  = note: expected unit type `()`
                found struct `queries::builders::ObjectArgumentBuilder<'_, schema::Args, ()>`
  = note: this error originates in the derive macro `cynic::QueryFragment` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0308`.
error: could not compile `idk` (bin "idk") due to previous error
```

Removing the `arguments` attribute allows the compilation to succeed.

--- a/cynic-codegen/src/fragment_derive/arguments/output.rs
+++ b/cynic-codegen/src/fragment_derive/arguments/output.rs
@@ -95,7 +95,7 @@ impl ToTokens for ArgumentValueTokens<'_> {
                 for item in items {
                     let inner = self.wrap_value(item);
                     tokens.append_all(quote! {
-                        .item(|builder| { builder #inner })
+                        .item(|builder| { builder #inner; })
                     })
                 }
             }

